src/utils/scrollbar.ts

```diff
@@ -1,15 +0,0 @@
-import Colors from '@styles/Colors';
-
-export const GlobalScrollbarStyle = `
```

Comment:
I'm actually using this within the new compare modal now so I have to see how I can use your new approach. Could we pair for a bit before merging to see if it works?